Utilities/WSIWindow/android/native.cpp

```diff
@@ -0,0 +1,127 @@
+#include "native.h"
+
+android_app* Android_App=0;                //Android native-actvity state
+/*
```

Comment:
The commented out function is a work in progress, and not ready yet.
A few Android event types are not being handled yet.